apidocs/getInclusionStates.md

```diff
@@ -0,0 +1,115 @@
+
+---
+### [getInclusionStates](https://github.com/iotaledger/iri/blob/dev/src/main/java/com/iota/iri/service/API.java#L747)
+ [AbstractResponse](/javadoc/com/iota/iri/service/dto/abstractresponse/) getInclusionStatesStatement(java.util.List trans, java.util.List tps)
+
+Get the inclusion states of a set of transactions. 
```

Comment:
This is for determining -> Let's you determine

Can we remove all occurrences of  'thus' from the description as well?
...transaction list you submitted, thus you get a -> ...transaction list you submitted. You get a